src/OpenTelemetry.Exporter.Zipkin/ZipkinExporterType.cs

```diff
@@ -0,0 +1,24 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
```

Comment:
Seems most of the using statements are not needed?

Reply:
Fixed this.